homeassistant/components/unifiprotect/media_source.py

```diff
@@ -393,19 +447,14 @@ async def _build_event(
         title = dt_util.as_local(start).strftime("%x %X")
         duration = end - start
         title += f" {_format_duration(duration)}"
-        if event_type == EventType.RING.value:
+        if event_type in EVENT_MAP[SimpleEventType.RING]:
```

Comment:
Not sure how often these are called but if it's often it would be better to make them constants to avoid doing the dict lookup in the event map for each one